### DISCUSSION TITLE
réinstallation complète sur Jeedom Luna - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#9] [Comments]
# réinstallation complète sur Jeedom Luna - Haade.fr

tuto complet sur une installation complète après plantage de jeedom basé sur debian sur Jeedom Luna

https://haade.fr/fr/blog/reinstallation-debian-jeedom-sur-jeedom-luna

**Comment by durix75** [comment]
Hello,
Merci pour ton super tuto
J'ai une DSGW 210 achetée directement chez dusun et il m'a bien servi
J'ai toutefois un probleme mon adresse MAC ethernet change à chaque reboot ce qui m'empeche de lui assigner une adresse IP statique.
Tu saurais comment désactiver ce comportement inhabituel?
Merci

**Reply by haade-administrator** [reply]
pour tes tests désactive le wifi et garde l'ethernet redémarre et dit moi si l'adresse mac change

**Reply by haade-administrator** [reply]
tu dois avoir un conflit sur le lan-wifi

**Reply by durix75** [reply]
J’ai bloqué la connection wifi avec rfkill. 
après reboot l adresse mac ethernet à encore changée
Merci de ton aide

**Reply by haade-administrator** [reply]
alors regarde du côté du routeur c'est lui qui fou le fouillis et alloue l'adresse mac essaie de paramétrer en ip fixe ta jeedom luna

**Comment by Silverman93** [comment]
Bonjour Nico

Merci pour ton article. Je voulais savoir si on pouvais faire l'inverse. 
C'est a dire transformer une LUNA officiel en Box HA?
Merci d'avance de ton retour
Stéphane

**Reply by haade-administrator** [reply]
Salut, si tu veux installer installer l'image Homeassistant sur Docker c'est pas compliqué, par contre je n'ai pas réussi à installer HAOS, Dusun ne m'a pas fourni l'image et en installation manuel je n'ai pas réussi à installer le supervised installer.

**Reply by haade-administrator** [reply]
tiens peut être que la manip fonctionne maintenant je te laisse tester

[Installer Haos sur Dusun](https://www.dusuniot.com/blog/how-to/how-to-install-application-on-dsgw-210/#elementor-toc__heading-anchor-4)

**Reply by Silverman93** [reply]
Merci pour ton retour rapide
Effectivement cela serais plus propre sous HAOS, mais je vais essayé sous docker. Dans les grande ligne on reprend le début de ton tuto et apres on installe HA sous docker comme expliqué ici : https://www.dusuniot.com/blog/how-to/how-to-install-application-on-dsgw-210/

**Reply by haade-administrator** [reply]
c'est ça utilise les mêmes outils, l'image debian est dispo à cette adresse: [https://drive.google.com/file/d/1NHFqo69C4M7LMB_wy1WnekQidKmkKJZh/view](https://drive.google.com/file/d/1NHFqo69C4M7LMB_wy1WnekQidKmkKJZh/view)

tu me diras si ça fonctionne

**Reply by Breizh13** [reply]
@Silverman93

Bonjour,

avez-vous réussi à installer HA sur la DUSUN DSGW-210? Si oui, j'aurais besoin d'un peu d'aide.

**Comment by Breizh13** [comment]
Bonsoir ou bonjour, c'est selon.

Bonsoir à tous, bonsoir Nico. Cette page m'a occupé quelques heures aujourd'hui en // de cette video YT https://www.youtube.com/watch?v=xN4Wk5sfmGE
Mon but, installer HA sur une DUSUN DSGW-210 (F18) achetée chez Digikey. N'arrivant pas à  installer correctement HA, je me suis allé jusqu'au bout de cette procédure et bingo, j'ai pu installer Jeedom, ce qui m'a d'une part rassuré et encouragé dans le sens ou je pense qu'on peut installer HA, reste à voir où je bugge! Merci pour pour ce tuto, merci pour les commentares encourageants. A bientôt pour des nouvelles ou alors pour encore un peu d'aide.
